Gordon360/Services/MembershipRequestService.cs

```diff
@@ -56,10 +56,15 @@ public async Task<RequestView> AddAsync(RequestUploadViewModel membershipRequest
 
         private bool RequestAlreadyExists(RequestUploadViewModel requestUpload)
         {
+            var acc = _accountService.GetAccountByUsername(requestUpload.Username);
+            if (acc == null) throw new UnprocessibleEntity() { ExceptionMessage = "Account not found" };
```

Comment:
This is redundant because `accountService.GetAccountByUsername` already performs this check.

Reply:
valid